### ISSUE TITLE
accessibility: timezone selection `input` doesn't have a `label`

### ISSUE THREAD

**Issue by mfranzke** [calcom/cal.com#17186]
### Issue Summary

The invisible `input`-element that's being used to store the timezone selection (CSS selector `.current-timezone input[role="combobox"]`) doesn't include an (at least invisible) `label`.

### Steps to Reproduce

1. Go to a book event form.
2. E.g. run Google Lighthouse out of Google Web Developer tools.

### Actual Results

You'll get a report that the `input` doesn't have an associated `label` currently.

### Expected Results

At best this label would be visible as well as most form elements should include a visible `label` according to [WCAG Success Criterion C 3.3.2](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html), but at least include this as a invisible one so that screenreaders could pick up that value.

### Technical details

independent from browser environment.

### Evidence

Go to _Accessibility_ sub-tab underneath the _Elements_ section within Google Developer tools after inspecting the `input` element and check for a textual description. Or just rerun Google Lighthouse to check for any reportings on this topic.

**Comment by Himanshu-Kathania** [comment]
Can you assign this issue to me ?

**Comment by chrhi** [comment]
.take

**Comment by shivambhatia604** [comment]
How di I reach book event form?

**Comment by shivambhatia604** [comment]
@mfranzke @anikdhabal Please review the PR for this issue.
https://github.com/calcom/cal.com/pull/17241
